app.css

```diff
@@ -0,0 +1,54 @@
+* {
+  box-sizing: border-box;
+  font-size: 10px;
+  font-family: sans-serif;
+  font-weight: bold;
+}
+
+body {
+  width: 900px;
+  margin: auto;
+  color: lightgray;
+}
+
+.cat-column {
+  float: left;
+  display: inline;
+  margin: auto;
+  width: 280;
+  padding: 10px;
+}
+
+.octocats {
+  float: left;
+  width: 275px;
+  padding: 25px;
+  border: lightgray 1px solid;
+  display: inline;
+
+}
+
+.number {
+  position: relative;
+  bottom: -12px;
+  padding: 8px 0 18px 0;
+
+}
+
+.created {
+  float: right;
+  display: inline inline-block;
+  position: relative;
```

Comment:
Position relative should only be used to make small "tweaks" like we discussed today in class. You should be able to achieve the full layout with just `display` and `float`.

Also check the above `display` property in Chrome inspect element and you'll see that neither display is taking effect. Display inline only displays within an element look at adding float to the `.number`.